src/todolist.js

```diff
@@ -0,0 +1,40 @@
+let todoList = localStorage.getItem('todoList') ? JSON.parse(localStorage.getItem('todoList')) : [];
+const getTodoList = () => todoList;
+const saveTodoList = () => {
+  localStorage.setItem('todoList', JSON.stringify(todoList));
+};
+const addItem = (task) => {
+  if (task) {
+    const taskItem = {
+      description: task,
+      completed: false,
+      index: todoList.length + 1,
+    };
+    todoList.push(taskItem);
+    saveTodoList();
+  }
+};
+
+const removeItem = (index) => {
+  todoList.splice(index, 1);
+  localStorage.setItem('todoList', JSON.stringify(todoList));
+};
```

Comment:
- [ ] After removing the task, the `index` values of the remaining tasks should be updated to reflect their positions in the array. If there are 5 tasks, and I remove anyone, regardless of position, the remaining `index` values should still be 1, 2, 3 and 4 respectively.